pkg/util/json/json.go

```diff
@@ -45,6 +45,24 @@ const (
 	ObjectJSONType
 )
 
+func (t Type) String() string {
+	switch t {
+	case NullJSONType:
+		return "null"
+	case StringJSONType:
+		return "string"
+	case NumberJSONType:
+		return "number"
+	case FalseJSONType, TrueJSONType:
+		return "boolean"
+	case ArrayJSONType:
+		return "array"
+	case ObjectJSONType:
+		return "object"
+	}
+	panic(fmt.Sprintf("Type %d needs to be taken into accout", int(t)))
```

Comment:
I think it's slightly more idiomatic (at least within the Cockroach codebase) to write this as "unknown type %d", and also I don't think the explicit cast to `int` is necessary? Though I might be wrong.